litmus-portal/graphql-server/pkg/chaos-workflow/handler/handler.go

```diff
@@ -490,7 +562,17 @@ func QueryListWorkflow(project_id string, workflowIds []*string) ([]*model.Workf
 		}
 		result = append(result, &newChaosWorkflows)
 	}
-	return result, nil
+
+	totalFilteredWorkflows := 0
```

Comment:
Please rename it as totalFilteredWorkflowsCounter.

Reply:
Done